src/scene/x3d/x3dnodes_standard_time.inc

```diff
@@ -1,3 +1,4 @@
+{%MainUnit X3DNodes.pas}
```

Comment:
Specify filenames lowercase here, as they are strict lowercase on Unix-like systems, so please make this

```
{%MainUnit x3dnodes.pas}
```